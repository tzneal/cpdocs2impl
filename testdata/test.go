package testdata

type Fooer interface {
	// Foo does some foo stuff
	Foo()
	// Bar is different
	Bar()
}

var _ Fooer = (*Concrete)(nil)

type Concrete struct {
}

func (c *Concrete) Foo() {}
func (c *Concrete) Bar() {}
