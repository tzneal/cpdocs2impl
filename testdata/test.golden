package testdata

type Fooer interface {
	// Foo does some foo stuff
	Foo()
	// Bar is different
	Bar()
}

var _ Fooer = (*Concrete)(nil)

type Concrete struct {
}


// Foo does some foo stuff
func (c *Concrete) Foo() {}
// Bar is different
func (c *Concrete) Bar() {}
